server/internal/adapter/publicapi/types.go

```diff
@@ -118,6 +124,18 @@
 			} else if len(itemAssets) > 0 {
 				val = itemAssets[0]
 			}
+		} else if sf.Type() == value.TypeGroup {
+			res := map[string]ItemFields{}
+			for _, v := range f.Value().Values() {
+				itgID, ok := v.ValueGroup()
+				if !ok {
+					continue
+				}
+				gf := fields.FieldsByGroup(itgID)
+				igf := NewItemFields(gf, groupFields, nil, assets, urlResolver)
+				res[itgID.String()] = igf
+			}
+			val = res
```

Comment:
multiple group is not handled correctly, check schema field to determine if the field is multiple or not.
output example for single:
```json
...
"groupKey": {
   "groupFieldKey": "groupFieldValue",
   ...
}
```

output example for multiple:
```json
...
"groupKey": [
   {
      "groupFieldKey": "groupFieldValue",
      ...
   },
   {}
]
```

Reply:
multiple fields should show like the following:
```
"groupFiledKey": {
    "itemGroupFieldID1": {
       "field1Key": "value",
       "field2Key": "value",
    },
    "itemGroupFieldID2": {
       "field1Key": "value",
       "field2Key": "value",
    },
    "itemGroupFieldID3": {
       "field1Key": "value",
       "field2Key": "value",
    }
  }
  ```
  I think it's important to show the itemGroup IDs as well, so a map is the better than an array in this case